### ISSUE TITLE
Rockpool time mis-labelled

### ISSUE THREAD

**Issue by ContraryCamber** [pimoroni/flotilla-rockpool#13]
In rockpool shipshape the Time input appears to incorrectly classify:
Seconds as Minutes
Minutes as Hours
Hours as Days

I.e. If you select Minutes, the output increments by the second

http://flotil.la/shipshape/

Looking at this I think the measures are showing relative portion of the current minute, hour or day as range between 0 - 1000, as opposed to the absolute hh:mm:ss value

Nb. I was trying to work out if I could create a digital clock, outputting Time to Number

**Comment by Gadgetoid** [comment]
They're singular, ie: Minute, Hour, Day. Representing slices of those time periods. However in retrospect the choice of nomenclature could be confusing- Seconds, Minutes and Hours may well be better.

**Comment by Gadgetoid** [comment]
I've adjusted the nomenclature to be less confusing.

I have also made some additions to the Number module- namely an Hour, Minute and Second output alongside the "Number" one. 
- Hour displays a scaled 24 hour time in the first 2 digits of Number, compatible with new newly named "Hours" input of time.
- Minute displays a scaled number of minutes in the second 2 digits of Number, compatible with the newly named "Minutes" input of time.
- Second toggles the colon on and off, `on = (input_value * 59) % 2`

This, with a simple set of rules, allows a full digital clock to be created on number. It should be possible to use comparisons and thresholds to make a functional alarm too :D
